src/command.ts

```diff
@@ -864,6 +864,17 @@ function buildEnv(env: Record<string, string | undefined>) {
   return result;
 }
 
+function getCwd(): string {
+  const readPermissions = Deno.permissions.querySync({
+    name: "read",
+    path: ".",
+  });
+  if (readPermissions.state == "denied") {
+    return "";
```

Comment:
Can you add a test to demonstrate this works? I think you can set permissions for an individual test by doing:

```ts
Deno.test({
  name: "...etc...",
  permissions: {
    read: false,
  },
}, async () => {

});
```

I'm unsure about this change. What happens if someone executes a `cd` command? What's the cwd of launched commands? Maybe this should instead be `undefined` and then the individual commands would then need to handle that.